server/bin/test-bin/test_logger_type.py

```diff
@@ -55,5 +55,5 @@ def test_pbench_logger():
     try:
         test_pbench_logger()
     except BadConfig as bd:
-        print("BadConfig exception was raised")
+        print(f"BadConfig exception was raised, '{bd}'")
```

Comment:
I think you would want to log this rather than the print no?

Reply:
We were printing this before, I just added the bad config exception to what was printed.  If we log it when we are testing the logger, we would loop, I think.